objax/module.py

```diff
@@ -22,7 +22,7 @@
 from jax.interpreters.pxla import ShardedDeviceArray
 
 from objax.typing import JaxArray
-from objax.util import positional_args_names
+from objax.util import merge_args_kwargs, positional_args_names
```

Comment:
See code comments below to see why I changed the function (it's not just a rename).
```suggestion
from objax.util import overridee_args_kwargs, positional_args_names
```